### DISCUSSION TITLE
All Wallets page cannot connect to any wallets and QR code won't load

### DISCUSSION THREAD

**Discussion by rubyad** [WalletConnect/walletconnect-monorepo#4192] [Developer Support]
On page load I'm getting this error in the console: 
`import_ethereum_provider.default.init is not a function
    at Web3Modal.initWalletConnectProvider`
Also the WalletConnect QR code doesn't load and clicking on any wallet in the All Wallets panel produces this error from `modal.subscribeEvents`:
`message: 'connectionControllerClient:getWalletConnectUri - provider is undefined'`
Would appreciate any ideas on how to fix, thanks

**Comment by glitch-txs** [comment]
Please share a minimal reproducible example of the issue
